services/monitor/instances/monitor_gci_group_settings/constraints/external_members_denied_on_secgroups/constraint.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: CIGroupSettingsConstraintV1
+metadata:
+  name: external_members_denied_on_mailinglists
+  annotations:
+    description: allowExternalMembers must be false for all groups.
+    category: Identity and Access Management
+spec:
+  severity: major
+  match:
+    target: ['directories/'] # to_be_adapted
```

Comment:
Why between single quotes?
For organizations it works without quotes

Reply:
corrected